src/components/layout/location-context.tsx

```diff
@@ -1,33 +1,30 @@
-import React, { ReactNode, Dispatch, SetStateAction } from 'react'
+import React, { Dispatch, SetStateAction, PropsWithChildren } from 'react'
 import type { ModalPayloadType } from './layout'
 
 type LocationProviderProps = {
-    children: ReactNode
     has_mounted?: boolean
-    setModalPayload?: Dispatch<SetStateAction<ModalPayloadType>>
     show_cookie_banner?: boolean
+    setModalPayload?: Dispatch<SetStateAction<ModalPayloadType>>
     toggleModal?: (event: React.MouseEvent<HTMLElement>) => void
 }
 
-export const LocationContext = React.createContext(true)
+export const LocationContext = React.createContext<LocationProviderProps>(null)
```

Comment:
@niloo-fs Would you make sure that changing the initial value to `null` won’t break anything 🙏🏻

Reply:
I don't think there would be any problem but don't worry I'll check this with QA too